internal/config/subnet/api-key.go

```diff
@@ -59,13 +73,28 @@ func validateLicenseFormat(lic string) error {
 	return err
 }
 
+func validateAPIKeyFormat(apiKey string) error {
+	if len(apiKey) == 0 {
+		return nil
+	}
+
+	// Verify that the api key is a valid UUID string
+	_, err := uuid.Parse(apiKey)
```

Comment:
```suggestion
	_, err := uuid.Parse(apiKey)
```

are you sure this won't change in the future to some other format? - lets not do what we did with JWT since that was for a different purpose.

Reply:
Since it is possible to set/get this using `mc admin config set|get` also, this was done for protection against setting incorrect value by mistake. But I see your point - it would create unnecessary problems if the format of the api key changes in future. I'll remove this validation.

Reply:
Removed the validation